tools/ci_connector_ops/ci_connector_ops/qa_checks.py

```diff
@@ -2,65 +2,106 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
+
 import sys
 
-def check_documentation_markdown_file(connector_name: str) -> bool:
+from ci_connector_ops.utils import Connector
+
+
+def check_documentation_file_exists(connector: Connector) -> bool:
     """Check if a markdown file with connector documentation is available 
-    in docs/integrations/<connector-type>/<connector-name>.md
+    in docs/integrations/<connector-type>s/<connector-name>.md
 
     Args:
-        connector_name (str): The connector name
+        connector (Connector): a Connector dataclass instance.
 
     Returns:
         bool: Wether a documentation file was found.
     """
-    # TODO implement
-    return True
 
-def check_changelog_entry_is_updated(connector_name: str) -> bool:
+    return connector.documentation_file_path.exists()
+
+def check_documentation_follows_guidelines(connector: Connector) -> bool:
+    """Documentation guidelines are defined here https://hackmd.io/Bz75cgATSbm7DjrAqgl4rw"""
```

Comment:
neat! I hadn't seen these guidelines. 

Are these linked to from any official airbyte site (e.g. docs?) If so, may be worth pointing to the airbyte link rather than hackmd in case it ever changes...

Reply:
@airbytehq/docs is https://hackmd.io/Bz75cgATSbm7DjrAqgl4rw making its way to our official doc?

Reply:
Nope, not yet at least. The hackmd format lets contributors use the markdown format more easily than the Docs site.

Reply:
I meant: is there a link to this template? Are these connectors doc guidelines shared in our official doc?

Reply:
We link to it from our Contribution guide: https://docs.airbyte.com/contributing-to-airbyte/updating-documentation#additional-guidelines
Is that what you're looking for?